topics/proteomics/tutorials/protein-id-oms/tours/tour.yaml

```diff
@@ -0,0 +1,590 @@
+      href="https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/database-handling/tutorial.html">Database
+      Handling</a> you can use the constructed database including decoys. You
+      can find a prepared database, as well as the input proteomics data in
+      different file formats on <a
+      href="https://zenodo.org/record/796184">Zenodo</a>.
+    backdrop: true
+  - title: History options
+    element: '#history-options-button'
+    content: >-
+      We will start the analyses by creating a new history. Click on this button
+      and then "Create New"
+    placement: left
+  - title: Preparing raw data
+    content: >-
+      Raw data conversion is the first step of any proteomic data analysis. The
+      most common converter is MSConvert, the format to convert to is mzML.
+      Search GUI takes only mgf format as input, but as we need the mzML format
+      for several other tasks, we will convert to mzML first. Due to licensing
+      reasons, MSConvert runs only on windows systems and will not work on most
+      Galaxy servers.
+    backdrop: true
+  - title: Preparing raw data
+    content: >-
+      Depending on your machine settings, raw data will be generated either in
+      profile mode or centroid mode. For most peptide search engines, the MS2
+      data have to be converted to centroid mode, a process called “peak
+      picking” or “centroiding”. Machine vendors offer algorithms to extract
+      peaks from profile raw data. This is implemented in <b>msconvert</b>  and
+      can be run in parallel to the mzML conversion. However, the OpenMS tool
+      <b>PeakPickerHiRes</b>  is reported to generate slightly better results
+      (<a href="https://www.ncbi.nlm.nih.gov/pubmed/17094243">Lange et al.,
+      2006, Pac Symp Biocomput</a>) and is therefore recommended for
+      quantitative studies (<a
+      href="https://www.ncbi.nlm.nih.gov/pubmed/19953549">Vaudel et al., 2010,
+      Proteomics</a>). If your data were generated on a low resolution mass
+      spectrometer, use <b>PeakPickerWavelet</b> instead.
+    backdrop: true
+  - title: Preparing raw data
+    content: >-
+      We provide the <a href="https://zenodo.org/record/796184">input data</a>
+      in the original raw format and also already converted to mgf and mzML file
+      formats. If <b>msconvert</b> does not run on your Galaxy instance, please
+      download the preconverted <b>mzML</b> as an input.
+    backdrop: true
+  - title: Uploading the new data
+    element: '#tool-panel-upload-button .fa.fa-upload'
+    content: We need to upload data. Open the Galaxy Upload Manager
+    placement: right
+    postclick:
+      - '#tool-panel-upload-button .fa.fa-upload'
+      - '#btn-reset'
+  - title: Uploading the input data
+    element: '#btn-new'
+    content: Click on Paste/Fetch Data
+    placement: right
+    postclick:
+      - '#btn-new'
+  - title: Uploading the input data
```

Comment:
Would it be possible to provide an already converted file? `msconvert` needs to run on Windows to support raw formats - which is probably not set up on most Galaxy instances (and especially not supported in docker flavors which could benefit from including this tutorial). But we should mention this hiccup to make users aware on how to handle their own data.

Reply:
A converted file is provided alongside the `raw` in Zenodo. We should definitely use the `mzML` file and skip the conversion! I can do the necessary changes tomorrow.